modules/vlybyBidAdapter.js

```diff
@@ -0,0 +1,76 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js'
+
+import { BANNER, VIDEO } from '../src/mediaTypes.js'
+import { config } from '../src/config.js';
+
+const ENDPOINT = '//europe-west3-vlybypoc2019.cloudfunctions.net/prebid';
+const BIDDER_CODE = 'vlyby';
```

Comment:
Is there a `GVL_ID` that can be associated with this  bidder? Not sure if this plays well with the `gdprEnforcement` module

Reply:
@muuki88 There is no Global Vendor List ID. Due to the fact, that we are not processing any user data. For gdprEnforcement. Every Implementation needs a provided user ID and contract between VLYBY and Publisher. So for GPDR we will communicate direct with Publisher. Maybe gdpr.rules[].vendorExceptions should be used in this case.

Reply:
As long as you don't add any user syncs this is probably fine.

However you are a registered vendor now, so why not use it?